input/pages/operations.md

```diff
@@ -0,0 +1,56 @@
+{: #operations}
+
+## Operations
+* [**[base]/$cqfm.package**](OperationDefinition-cqfm-package.html)
+* [**Library/$cqfm.package**](OperationDefinition-cqfm-package.html)
+* [**Library/[id]/$cqfm.package**](OperationDefinition-cqfm-package.html)
+* [**Library/$data-requirements**](OperationDefinition-Library-data-requirements.html)
+* [**Measure/[id]/$cqfm.package**](OperationDefinition-cqfm-package.html)
+* [**Measure/$data-requirements**](OperationDefinition-Measure-data-requirements.html)
+* [**MeasureReport/$cqfm.package**](OperationDefinition-cqfm-package.html)
+* [**MeasureReport/[id]/$cqfm.package**](OperationDefinition-cqfm-package.html)
+* [**ValueSet/$expand**](OperationDefinition-ValueSet-expand.html)
+
+
+#### Artifact Repository Operations
```

Comment:
These aren't really FHIR operations, they are just descriptions of the operations that an artifact repository must support. Please remove this section.